_data/master/navbars/usage.yml

```diff
@@ -17,6 +17,10 @@ toc:
       path: /usage/calicoctl/configure/etcd
     - title: Kubernetes API datastore options
       path: /usage/calicoctl/configure/kdd
+- title: Using Calico network policy 
```

Comment:
I guess this is a section we plan to add more content to later on, although at this time, it only contains the one item. I would suggest maybe we don't need a section just yet, with only the one item inside. As long as the paths don't change, it's very easy to make it a section later.